app/assets/stylesheets/static_pages.css.scss

```diff
@@ -1,3 +1,6 @@
-// Place all the styles related to the static_pages controller here.
-// They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
+html, body, #map-canvas{
```

Comment:
html and body should be 100%! With this setting, #map-canvas takes 75%*75%*75% = 42% of the screen height!